packages/blockbook/src/controller.ts

```diff
@@ -642,15 +642,6 @@ export class Blockbook extends Controller {
    * @example account "14mMwtZCGiAtyr8KnnAZYyHmZ9Zvj71h4t"
    * @example account "xpub6DQYbVJSVvJPzpYenir7zVSf2WPZRu69LxZuMezzAKuT6biPcug6Vw1zMk4knPBeNKvioutc4EGpPQ8cZiWtjcXYvJ6wPiwcGmCkihA9Jy3"
    */
-  @Example<Array<Utxo>>([
```

Comment:
And this is still a valid response for utxo by address

Reply:
yep all UtxoResponse is pulled out